libqtile/widget/backlight.py

```diff
@@ -84,7 +86,7 @@ def poll(self):
         return self.format.format(percent=percent)
 
     def change_backlight(self, value):
-        self.call_process(["xbacklight", "-set", str(value)])
+        self.call_process(shlex.split(self.change_command % value))
```

Comment:
Any reason to not use `.format` and its templating rather than `%`?

Reply:
refactored to use `format` method